pkgs/java_http/lib/src/java_client.dart

```diff
@@ -65,9 +65,8 @@ class JavaClient extends BaseClient {
     });
 
     final contentLengthHeader = responseHeaders['content-length'];
-    final contentLength = (contentLengthHeader == null)
-        ? null
-        : int.tryParse(contentLengthHeader);
+    final contentLength =
```

Comment:
:heavy_check_mark: Done! 

The code now throws a `ClientException` instead of a `FormatException` if the content length is not an integer.

I ended up using a slightly different approach to the code in "Alternative Solution 1".
I also put the code for getting the content length header into its own method.

Thank you!